src/main/scala/io/iohk/ethereum/consensus/pow/EthashMiner.scala

```diff
@@ -0,0 +1,205 @@
+package io.iohk.ethereum.consensus
+package pow
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Props}
```

Comment:
What is the sentiment of the team about Typed Actor?

Reply:
In ETCM-759 I am adding 2 new actors, that will be typed, but for now the decision has been to do new things with Typed Actors and not migrate the Classic ones. Maybe in the end this one will also be migrated